### PR TITLE
 Problem: tx log attribute value is not parsable by some clients

### PR DESCRIPTION
**Pull request by yihuang**
Solution:
- update ethermint dependency

👮🏻👮🏻👮🏻 !!!! REFERENCE THE PROBLEM YOUR ARE SOLVING IN THE PR TITLE AND DESCRIBE YOUR SOLUTION HERE !!!! DO NOT FORGET !!!! 👮🏻👮🏻👮🏻

# PR Checklist:

- [ ] Have you read the [CONTRIBUTING.md](https://github.com/crypto-org-chain/chain-main/blob/master/CONTRIBUTING.md)?
- [ ] Does your PR follow the [C4 patch requirements](https://rfc.zeromq.org/spec:42/C4/#23-patch-requirements)?
- [ ] Have you rebased your work on top of the latest master? 
- [ ] Have you checked your code compiles? (`make`)
- [ ] Have you included tests for any non-trivial functionality?
- [ ] Have you checked your code passes the unit tests? (`make test`)
- [ ] Have you checked your code formatting is correct? (`go fmt`)
- [ ] Have you checked your basic code style is fine? (`golangci-lint run`)
- [ ] If you added any dependencies, have you checked they do not contain any known vulnerabilities? (`go list -json -m all | nancy sleuth`)
- [ ] If your changes affect the client infrastructure, have you run the integration test?
- [ ] If your changes affect public APIs, does your PR follow the [C4 evolution of public contracts](https://rfc.zeromq.org/spec:42/C4/#26-evolution-of-public-contracts)?
- [ ] If your code changes public APIs, have you incremented the crate version numbers and documented your changes in the [CHANGELOG.md](https://github.com/crypto-org-chain/chain-main/blob/master/CHANGELOG.md)?
- [ ] If you are contributing for the first time, please read the agreement in [CONTRIBUTING.md](https://github.com/crypto-org-chain/chain-main/blob/master/CONTRIBUTING.md) now and add a comment to this pull request stating that your PR is in accordance with the [Developer's Certificate of Origin](https://github.com/crypto-org-chain/chain-main/blob/master/CONTRIBUTING.md#developer-certificate-of-originn).

Thank you for your code, it's appreciated! :)

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -36,6 +36,8 @@ replace github.com/99designs/keyring => github.com/crypto-org-chain/keyring v1.1
 replace github.com/cosmos/ibc-go => github.com/crypto-org-chain/ibc-go v1.0.1-hooks
 
 // FIXME: update after PR merged: https://github.com/PeggyJV/gravity-bridge/pull/182
-replace github.com/peggyjv/gravity-bridge/module => github.com/crypto-org-chain/gravity-bridge/module v0.1.22-0.20210923075944-630e7235f4b2
+replace github.com/peggyjv/gravity-bridge/module => github.com/yihuang/gravity-bridge/module v0.1.22-0.20211003003616-466845e76fcf
 
 replace github.com/cosmos/iavl => github.com/cosmos/iavl v0.17.1
+
+replace github.com/tharsis/ethermint => github.com/yihuang/ethermint v0.4.2-0.20211003002300-c53ebc8bbdaf
```
Comment (thomas-nguy) [inline go.mod:43]: which version of ethermint does it uses?

Comment (yihuang) [reply go.mod:43]: the exact commit in this PR: https://github.com/tharsis/ethermint/pull/615

File: go.mod [hunk 2]
```diff
@@ -36,6 +36,8 @@ replace github.com/99designs/keyring => github.com/crypto-org-chain/keyring v1.1
 replace github.com/cosmos/ibc-go => github.com/crypto-org-chain/ibc-go v1.0.1-hooks
 
 // FIXME: update after PR merged: https://github.com/PeggyJV/gravity-bridge/pull/182
-replace github.com/peggyjv/gravity-bridge/module => github.com/crypto-org-chain/gravity-bridge/module v0.1.22-0.20210923075944-630e7235f4b2
+replace github.com/peggyjv/gravity-bridge/module => github.com/yihuang/gravity-bridge/module v0.1.22-0.20211003003616-466845e76fcf
```
Comment (yihuang) [inline go.mod:39]: we should merge the recent main branch of gravity-bridge into the Cronos one: https://github.com/PeggyJV/gravity-bridge/pull/211

File: go.mod [hunk 3]
```diff
@@ -36,6 +36,8 @@ replace github.com/99designs/keyring => github.com/crypto-org-chain/keyring v1.1
 replace github.com/cosmos/ibc-go => github.com/crypto-org-chain/ibc-go v1.0.1-hooks
 
 // FIXME: update after PR merged: https://github.com/PeggyJV/gravity-bridge/pull/182
-replace github.com/peggyjv/gravity-bridge/module => github.com/crypto-org-chain/gravity-bridge/module v0.1.22-0.20210923075944-630e7235f4b2
+replace github.com/peggyjv/gravity-bridge/module => github.com/yihuang/gravity-bridge/module v0.1.22-0.20211004002640-008106caa0eb
```
Comment (yihuang) [inline go.mod:39]: this is the version in this PR: https://github.com/crypto-org-chain/gravity-bridge/pull/13

File: go.mod (hunk 1 above)
Comment (thomas-nguy) [reply go.mod:43]: can you replace it by `crypto.org/ethermint` ? It would be nice also to tag the version we wants to patch

**Comment by yihuang** [comment]
Closing, superseded by https://github.com/crypto-org-chain/cronos/pull/134

**Pull request closed without merging**